test/TestProjects/Spread-Cadl/Spread-Cadl.cadl

```diff
@@ -34,8 +34,60 @@ model Thing {
   age: int32;
 }
 
+alias ThingAlias = {
+  @doc("name of the Thing")
+  name: string;
+  @doc("age of the Thing")
+  age: int32;
+};
+
+alias ThingAliasWithModel = {
+  @path id: string;
+  ...Thing;
+  @header top: int32;
+};
+
+alias ThingMultiTargetAlias = {
+  @path id: string;
+  @doc("name of the Thing")
+  name: string;
+  @doc("age of the Thing")
+  age: int32;
+  @header top: int32;
+};
+
+alias ThingAliasWithSpreadAlias = {
+  @path id: string;
+  ...ThingAlias;
+  @header top: int32;
+};
+
 @route("/spreadModel")
 @doc("body parameter without body decorator")
 @post
 @convenienceAPI
-op spreadModel( ...Thing ): void;
\ No newline at end of file
+op spreadModel( ...Thing ): void;
+
+@route("/spreadAlias")
+@doc("body parameter without body decorator")
+@post
+@convenienceAPI
+op spreadAlias( ...ThingAlias ): void;
```

Comment:
Is this a valid case?  

op spreadAlias( ...ThingAlias, ...ThingMultiTargetAlias): void;

Reply:
This may impact how you name the internal model for alias.

Reply:
I think the behavior of `op spreadAlias( ...ThingAlias, ...ThingMultiTargetAlias): void; ` is the same as `op spreadAlias( ...ThingAlias ): void`